src/foraging_gui/Foraging.py

```diff
@@ -59,11 +59,7 @@ def __init__(self, parent=None,tower_number=1):
 
         # Load Laser and Water Calibration Files
         self._GetLaserCalibration()
-        try:
-            self._GetWaterCalibration()
-            logging.info('Loaded Water Calibration')
-        except Exception as e:
-            logging.error('Could not load water calibration file: {}'.format(str(e)))
+        self._GetWaterCalibration()
```

Comment:
Why do we need to run self._GetWaterCalibration() two times?

Reply:
It just runs once. The first call on line 61 is for GetLaserCalibration